### ISSUE TITLE
Enforce "indent" for Layout/MultilineOperationIndentation

### ISSUE THREAD

**Issue by mcary** [testdouble/standard#83]
I generally favor coding styles that reduce diff noise, and so wanted to suggest switching the Layout/MultilineOperationIndentation cop to enforce style "indented".  (It's presumably [enabled](https://github.com/testdouble/standard/blob/master/config/base.yml#L166-L167) with the [default of "aligned"](https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml#L867).)

The [examples in rubocop](https://github.com/rubocop-hq/rubocop/blob/092d7ea030d4828863b35cef1b287bca0476cda9/lib/rubocop/cop/layout/multiline_operation_indentation.rb#L9-L33) source show the styles in the context of an if-condition, which IMO obscures this case which is more common in the code I see:

Aligned:
```Ruby
foo = alpha +
      beta +
      gamma
```

Indented:
```Ruby
foo = alpha +
  beta +
  gamma
```

Now consider what happens if someone submits a PR changing `foo` to `foobar`.  With the "aligned" style, the 2nd and 3rd lines must be reindented, adding noise to the diff:

Aligned:
```Diff
-foo = alpha +
-      beta +
-      gamma
+foobar = alpha +
+         beta +
+         gamma
```

Where as with "indented", they do not:

Indented:
```Diff
-foo = alpha +
+foobar = alpha +
   beta +
   gamma
```

I know that options exist to make `diff` ignore whitespace, and I know that fancy diff binaries exist that can highlight only the changed substring within a line.  But this is not just about displaying patches to humans; it impacts Git's resolution of merge conflicts and commutability of patches with `git rebase -i` also.  The "indented" version would not cause merge conflict with a change immediately following the multi-line assignment, whereas the "aligned" version would.

This is also consistent with the "indented" setting for [MultilineMethodCallIndentation](https://github.com/testdouble/standard/blob/master/config/base.yml#L159-L161).

**Comment by searls** [comment]
I didn't even know about this one, but it's definitely more consistent! I like it

**Comment by searls** [comment]
Landed in 0.0.32
